config/clients/python/template/oauth2_test_sync.mustache

```diff
@@ -97,3 +97,145 @@ class TestOAuth2Client(IsolatedAsyncioTestCase):
         with self.assertRaises(AuthenticationError):
             client.get_authentication_header(rest_client)
         rest_client.close()
+
+    @patch.object(rest.RESTClientObject, "request")
+    async def test_get_authentication_obtain_with_expired_client_credentials_failed(
+        self, mock_request
+    ):
+        """
+        Expired token should trigger a new token request
+        """
+
+        response_body = """
+{
+  "reason": "Unauthorized"
+}
+        """
+        mock_request.return_value = mock_response(response_body, 403)
+
+        credentials = Credentials(
+            method="client_credentials",
+            configuration=CredentialConfiguration(
+                client_id="myclientid",
+                client_secret="mysecret",
+                api_issuer="www.testme.com",
```

Comment:
```suggestion
                api_issuer="https://tokenissuer.{{sampleApiDomain}}",
```

Reply:
or `https://issuer.{{sampleApiDomain}}`, both work. Just to remain consistent with our other SDKs, in general all outgoing requests are part of "{{sampleApiDomain}}", which is currently set to `fga.example`, which even if we set mocking incorrectly will never resolve to an actual domain

Reply:
Sure thing, I was just going off of what the existing Python tests were using — you good for me to just update them all in this PR, or would you prefer I cut a separate one?

Reply:
I'm fine with either, but if this PR is easier, let's do that

Reply:
Sounds good; PR updated 😄